templates/grafana/030-Deployment-grafana.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: enmasse
+  name: grafana
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: grafana
+    spec:
+      containers:
+      - name: grafana
+        image: grafana/grafana:latest
```

Comment:
Make it configurable using ${GRAFANA_IMAGE} and change Makefile to perform substitutiton.